### PR TITLE
Enable GitHub Release Zone in Main Branches.

### PR DESCRIPTION
**Pull request by thsparks**
This is a very targeted fix. I tried going through and making "main" and "master" equivalent throughout the github integration code ([thsparks/fix_main_in_github_integration](https://github.com/microsoft/pxt/tree/thsparks/fix_main_in_github_integration)), but I struggled with how to determine which one is used by a repo when on a non-main/master branch and I didn't think it was worth spending a ton of time on, so I paused that and focused on this quick fix instead. At least now, you can push releases and create branches from a Main branch.

### CODE REVIEW

**Review by srietkerk — APPROVED** [review]
Nice!!

File: webapp/src/gitjson.tsx
Comment (abchatra) [inline webapp/src/gitjson.tsx:239]: There is a _default_branch_ in github repo which you should query and do this?

Comment (abchatra) [reply webapp/src/gitjson.tsx:239]: _default_branch_ is configurable and need not be main.

Comment (abchatra) [reply webapp/src/gitjson.tsx:239]: Feel free to be not blocked on this.

Comment (thsparks) [reply webapp/src/gitjson.tsx:239]: Yeah this is not a fool-proof fix by any means. It's just a targeted change to unblock a bit of functionality. I think making us more resilient to different default branches would be nice, but a larger change.

Comment (thsparks) [reply webapp/src/gitjson.tsx:239]: Filed this as a follow-up, though I expect it's somewhat low-pri: https://github.com/microsoft/pxt/issues/9744

**Pull request merged**
